src/main/java/com/hermesworld/ais/galapagos/notifications/impl/NotificationServiceImpl.java

```diff
@@ -47,33 +47,18 @@ public class NotificationServiceImpl implements NotificationService {
 
     private final ITemplateEngine templateEngine;
 
-    private final InternetAddress fromAddress;
-
-    private final List<InternetAddress> adminMailRecipients;
+    private final GalapagosMailConfig galapagosMailConfig;
 
     public NotificationServiceImpl(SubscriptionService subscriptionService, ApplicationsService applicationsService,
             TopicService topicService, JavaMailSender mailSender, TaskExecutor taskExecutor,
-            @Qualifier("emailTemplateEngine") ITemplateEngine templateEngine,
-            @Value("${galapagos.mail.sender:Galapagos <me@privacy.net>}") String fromAddress,
-            @Value("${galapagos.mail.admin-recipients:}") String adminMailRecipients) {
+            @Qualifier("emailTemplateEngine") ITemplateEngine templateEngine, GalapagosMailConfig galapagosMailConfig) {
         this.subscriptionService = subscriptionService;
         this.applicationsService = applicationsService;
         this.topicService = topicService;
         this.mailSender = mailSender;
         this.taskExecutor = taskExecutor;
         this.templateEngine = templateEngine;
-        try {
-            this.fromAddress = InternetAddress.parse(fromAddress)[0];
-        }
-        catch (AddressException e) {
-            throw new RuntimeException("Invalid e-mail address specified as Galapagos FROM address", e);
-        }
-        try {
-            this.adminMailRecipients = toRecipientsList(adminMailRecipients);
```

Comment:
Please check if `toRecipientsList` and `safeToRecipientsList` are now still used, and remove them if not.